src/components/PlaceholderContainer/PlaceholderContainerAction/PlaceholderContainerAction.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import {Button, ButtonSize, ButtonView} from '@gravity-ui/uikit';
+
+import {block} from '../../utils/cn';
+
+import './PlaceholderContainerAction.scss';
+
+const b = block('placeholder-container-action');
```

Comment:
This should be the same block `placeholder-container` and action is an element of the block